### PR TITLE
Support Redmine 3.2.x and 3.3.x.

### PR DESCRIPTION
**Pull request by phlegx**

### CODE REVIEW

File: init.rb [hunk 1]
```diff
@@ -43,11 +43,31 @@
...
+end
+
 Redmine::Search.register :equipment_assets
 
-# Found redmine-1.4 -> redmine-2.1 porting info at http://www.redmine.org/boards/3/topics/30423
-#require 'dispatcher'
 Rails.configuration.to_prepare do
   require_dependency 'search_controller'
-  SearchController.send(:include, RedmineEquipmentStatusViewer::Patches::SearchControllerPatch)
+  # SearchController.send(:include, RedmineEquipmentStatusViewer::Patches::SearchControllerPatch)
```
Comment (sukima) [inline init.rb]: Is this required?

Comment (sukima) [inline init.rb]: @phlegx can this be merge with the patch or is Redmine now doing that automatically? What is the effect of commenting this line out? Once answered I can merge.

**Comment by phlegx** [comment]
@sukima solved all bugs. Try it out!

**Comment by sukima** [comment]
@phlegx I'll merge tomorrow.

For reference the commits were a bit all over the place. You might find this tip helpful: https://www.freshconsulting.com/atomic-commits/

**Pull request merged**